hw/xbox/nv2a.c

```diff
@@ -4496,6 +4674,11 @@ static void pgraph_method(NV2AState *d,
         pg->constants[59].dirty = true;
         break;
 
+    case NV097_SET_EYE_POSITION ...
+            NV097_SET_EYE_POSITION + 12:
```

Comment:
It is also declared as a four component vector. I don't remember why right now, but I'd assume this might go through some matrix or is normalized somehow (or w is simply ignored).